features/steps/cookies.py

```diff
@@ -84,26 +84,26 @@ def click_dropdown(context):
     )
 
 
-@when("I change my settings to enable cookies")
-def enable_cookie(context):
+@when('I change my settings to enable "{cookie_type}" cookies')
+def enable_cookie(context, cookie_type):
     WebDriverWait(driver, timeout=timeout).until(
-        lambda d: d.find_element(By.ID, "on-3-label")
+        lambda d: d.find_element(By.ID, f"on-{cookie_type}")
     ).click()
     WebDriverWait(driver, timeout=timeout).until(
-        lambda d: d.find_element(By.ID, "ons_cookie_save_changes")
+        lambda d: d.find_element(By.ID, "submit")
     ).click()
     WebDriverWait(driver, timeout=timeout).until(
         EC.presence_of_element_located((By.ID, "alert"))
     )
 
 
-@when("I change my settings to disable cookies")
-def disable_cookie(context):
+@when('I change my settings to disable "{cookie_type}" cookies')
+def disable_cookie(context, cookie_type):
     WebDriverWait(driver, timeout=timeout).until(
-        lambda d: d.find_element(By.ID, "off-3-label")
+        lambda d: d.find_element(By.ID, f"off-{cookie_type}")
     ).click()
     WebDriverWait(driver, timeout=timeout).until(
-        lambda d: d.find_element(By.ID, "ons_cookie_save_changes")
+        lambda d: d.find_element(By.ID, "submit")
```

Comment:
Why is there a ID change here?

Reply:
The new cookies.html has a different style of button. The 'type' of this button was labelled submit but did not have an ID. I added an Id and kept it consistent with the type but happy to change it now :)

Reply:
Done